images/win/scripts/Tests/Validate-Java.ps1

```diff
@@ -0,0 +1,50 @@
+################################################################################
+##  File:  Validate-Java.ps1
+##  Desc:  Validate Java
+################################################################################
+
+Import-Module "./Utils.psm1" -DisableNameChecking
+
+$JAVA_VERSIONS = @(7, 8, 11, 13)
+
+Describe "Java" {
+    # Checking the default JAVA 
+    It "Java 8 is default" {
+        $java8Path = Get-EnvironmentVariable "JAVA_HOME_8_X64"
+        $javaHomePath = Get-EnvironmentVariable "JAVA_HOME"
+        $java8Path | Should -Be "$javaHomePath"
+    }
+    
+    # Checking that Java and pluging are in PATH.
+    It "Java, Gradle, Maven, Ant added to the PATH:" {
+        if((Get-Command -Name 'java') -and (Get-Command -Name 'mvn') -and (Get-Command -Name 'ant') -and (Get-Command -Name 'gradle'))
+        {
+            Write-Host "Java $(java -version) on path"
+            Write-Host "Maven $(mvn -version) on path"
+            Write-Host "Ant $(ant -version) on path"
+            Write-Host "Gradle $(gradle -version) on path"
+        }
+        else
+            {
+            Write-Host "one of Java,Maven,Ant,Gradle is not on path."
+            exit 1
```

Comment:
We must not do `exit 1` in Pester tests. This command will cause script failure but the idea of Pester tests is running all test cases from file and then report summary.
Also every `It` block must contain at least one `Should` statement. And ideally, every `It` should test only one thing.

I suggest to rework `It "Java, Gradle, Maven, Ant added to the PATH:"` to something like that:
```
@("java", "gradle", "maven", "ant") | ForEach-Object {
  It "$_" {
    Validate-ZeroExitCode "$($_) --version" | Should -Be 0
  }
}
```